app/helpers/git_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GitHelper
```

Comment:
This looks like it should be a commit that reads more like

```Git
Generated git controller and routes

Ran 'rails g …
```

The current commit message is not helpful. It should explain what happened `Generate` and how that happened `ran …`